From 5dcc06a9607c87a79a442809c49c9e9556e7372b Mon Sep 17 00:00:00 2001
From: Peter Crosthwaite <peter.crosthwaite@xilinx.com>
Date: Mon, 29 Oct 2012 11:34:39 +1000
Subject: [PATCH 21/26] usb/ehci: Guard definition of EHCI_DEBUG

qemu mainline upstream commit 569c7fc8400ea3a7b13589af889003208aaae2ed

Guard against re-definition of EHCI_DEBUG. Allows for turning on of debug info
from configure (using --qemu-extra-cflags="-DEHCI_DEBUG=1") rather than source
code hacking.

Signed-off-by: Peter Crosthwaite <peter.crosthwaite@xilinx.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Integrated-by: Liming Wang <liming.wang@windriver.com>
---
 hw/usb/hcd-ehci.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hw/usb/hcd-ehci.c b/hw/usb/hcd-ehci.c
index 0775b5a..3f0d173 100644
--- a/hw/usb/hcd-ehci.c
+++ b/hw/usb/hcd-ehci.c
@@ -36,7 +36,9 @@
 #include "dma.h"
 #include "sysemu.h"
 
+#ifndef EHCI_DEBUG
 #define EHCI_DEBUG   0
+#endif
 
 #if EHCI_DEBUG
 #define DPRINTF printf
-- 
1.7.9.5

